common/store.ts

```diff
@@ -56,10 +59,30 @@ const configureStore = () => {
     'customTokens'
   );
 
+  const savedConfigState = loadStatePropertyOrEmptyObject<ConfigState>(
+    'config'
+  );
+
+  // If they have a saved node, make sure we assign that too. The node selected
+  // isn't serializable, so we have to assign it here.
+  if (savedConfigState && savedConfigState.nodeSelection) {
+    const savedNode = getNodeConfigFromId(
+      savedConfigState.nodeSelection,
+      savedConfigState.customNodes,
+    );
+    // If we couldn't find it, revert to defaults
+    if (savedNode) {
+      savedConfigState.node = savedNode;
+    }
+    else {
+      savedConfigState.nodeSelection = configInitialState.nodeSelection;
+    }
+  }
```

Comment:
This function's getting to be a bit of a doozy, maybe we could formalize loading from local storage into each reducer.